brave/brave-ios
# Fix #4831: Add 1.34 translations.

## Summary of Changes


This pull request fixes #4831 

## Submitter Checklist:


## Test Plan:

- Install using German language
- Open app, go to 3dot menu
- Verify that next to wallet `beta` label displays as `Betaversion` in German language

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
